Makefile

```diff
@@ -39,11 +39,11 @@ IMAGE_TAG := $(VERSION)
 IMAGE_ARCHS := amd64 arm arm64 386 # supported architectures
 
 GO = go
-GO_VERSION := 1.16
+GO_VERSION := 1.19
 LDFLAGS = -ldflags "-buildid= -s -w -X main.Version=$(VERSION) -X main.Revision=$(REVISION)"
 GO_BUILD = $(GO) build -tags="netgo" -trimpath $(LDFLAGS)
 UPX=upx --quiet --quiet
-DOCKER = DOCKER_CLI_EXPERIMENTAL=enabled DOCKER_BUILDKIT=1 docker
+DOCKER = DOCKER_CLI_EXPERIMENTAL=enabled DOCKER_BUILDKIT=1 sudo docker
```

Comment:
I am not sure. It seems to be working without `sudo`, when I tried to build the image locally.

As far as I know, @ailox introduced this change. Maybe he can say, if it is necessary here?

Reply:
No, `sudo docker` is only required if the executing user is not member of the "docker" group, but on the build machine this is not a problem. I think it may have been set for debugging, and was left in there. I think if we can do without sudo, it would be better.

The commit that introduced it was https://github.com/ubirch/ubirch-client-go/commit/d1b8657b4eda359ea1378589fcf5c55d3cfd02bc.

Reply:
```suggestion
DOCKER = DOCKER_CLI_EXPERIMENTAL=enabled DOCKER_BUILDKIT=1 docker
```